.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Build and release
+
+on:
+  push:
+    # Pattern matched against refs/tags
+    tags:
+      - '*' # Push events to every tag not containing /
+
+    # Allows us to run the workflow manually from the Actions tab
+    workflow_dispatch:
+
+env:
+    flutter_version: 3.13.6
+
+jobs:
+    chrome-extension-build-and-upload:
+      runs-on: ubuntu-22.04
+      steps:
+        - name: Checkout repository
+          uses: actions/checkout@v4
+        - name: Setup Flutter
+          uses: subosito/flutter-action@v2
+          with:
+            flutter-version: ${{ env.flutter_version }}
+            channel: 'stable'
+            cache: true
+        - name: Download pub dependencies
+          run: flutter pub get
+        - name: Build Extension
+          run: flutter build web --web-renderer html --csp --release
+        - name: Upload Bundle
+          uses: actions/upload-artifact@v3
+          with:
+            name: concordium-wallet-web-${{ github.ref_name }}
```

Comment:
Is the idea that you will always build from a tag, and then the version would be used here? I think it would be better to use the actual version from the application to avoid a mismatch between the build artifact and the actual application.

Reply:
The name of the artifact now seems to become:
```
concordium-wallet-web-1.0.0+1
```
Where does the `+1` come from?